src/main.c

```diff
@@ -1319,6 +1324,10 @@ main(int argc, char *argv[])
         np2_stderr_log = 0;
     }
 
+    /* use syslog even if running in foreground */
+    if (use_syslog == 1)
+        np2_stderr_log = 0;
```

Comment:
We do not allow `if` without braces in our coding style.

Reply:
Fixed in updated PR (force-push)